Source/DafnyDriver/DafnyBackwardsCompatibleCli.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Dafny;
 public interface ILegacyParseArguments { }
 
 // TODO: Refactor so that non-errors (NOT_VERIFIED, DONT_PROCESS_FILES) don't result in non-zero exit codes
-public enum ExitValue { SUCCESS = 0, PREPROCESSING_ERROR, DAFNY_ERROR, COMPILE_ERROR, VERIFICATION_ERROR, FORMAT_ERROR }
+public enum ExitValue { SUCCESS = 0, PREPROCESSING_ERROR = 1, DAFNY_ERROR = 2, COMPILE_ERROR = 3, VERIFICATION_ERROR = 4, FORMAT_ERROR = 5 }
```

Comment:
Ah does C# not guarantee this indexing by default?

Reply:
It does, but sometimes you have to know what the numbers are when looking at test output, so I found this easier to read.